docs/modules/ROOT/examples/tutorial/minio/deployment.yaml

```
@@ -26,6 +26,8 @@ spec:
         args:
         - server
         - /data
+        - '--console-address'
+        - ':9001'
```

I would keep it one argument, provided minio understands the `--arg=value` syntax (it probably does when using somewhat decent cli framework)
```suggestion
        - '--console-address=:9001'
```

---

according to [the documentation](https://docs.min.io/minio/baremetal/console/minio-console.html#static-vs-dynamic-port-assignment) and the warning output if not set, it is separated with a space:

```minio server https://minio-{1...4}.example.net/mnt/disk-{1...4} \
      --console-address ":9001"```

therefore I would leave it like that

---

looks like it works, after the change it's logged:
```
$ kubectl logs minio-75cb45d7fb-vszxv 
Finished loading IAM sub-system (took 0.0s of 0.0s to load data).
API: http://172.17.0.6:9000  http://127.0.0.1:9000 

Console: http://172.17.0.6:9001 http://127.0.0.1:9001 

Documentation: https://docs.min.io
```

Will push change.

---

https://github.com/k8up-io/k8up/pull/666/commits/253b1110558fd4b05a927849aa26f3238a13f9f3